Core/Debugger/WebSocket/HLESubscriber.cpp

```diff
@@ -480,9 +493,42 @@ void WebSocketHLEFuncScan(DebuggerRequest &req) {
 	if (!req.ParamU32("size", &size))
 		return;
 
+	bool has_recreate = req.HasParam("recreate");
+	bool recreate = false;
+	if (has_recreate) {
+		if (!req.ParamBool("recreate", &recreate)) {
+			return;
+		}
+	}
```

Comment:
You can replace this part with:

```c++
	bool recreate = false;
	if (!req.ParamBool("recreate", &recreate, DebuggerParamType::OPTIONAL))
		return;
```

In this case, because you treat not specified the same as false, there's no need to separately check existence.

-[Unknown]

Reply:
Right, of course. My job was to make it work and see if people with more experience can make it cleaner.

Reply:
Although I must admit that I used the existing code as a reference. Please check this: https://github.com/hrydgard/ppsspp/blob/master/Core/Debugger/WebSocket/BreakpointSubscriber.cpp#L270

Reply:
In that case, enabled on the breakpoint only changes if specified.  If omitted, it stays in its current state (which might be true or false.)  `hasEnabled` was required to do this because the current interface auto-enables things when you modify them.

-[Unknown]